gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageTest.java

```diff
@@ -1637,7 +1637,7 @@ public void googleCloudStorageItemInfo_toString() throws IOException {
     assertThat(itemInfo2.toString()).contains("exists: no");
   }
 
-  static <K, V> void assertMapsEqual(
+  public static <K, V> void assertMapsEqual(
```

Comment:
Why is this required? I could not see this getting used?

Reply:
This is used here - https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/1095/files#diff-603a2524bd5e4d31fea4c3596b45056b63dfa96f351d6d6816d4a06f0e09daddR556